adapters/silverpush/devicetype.go

```diff
@@ -0,0 +1,44 @@
+package silverpush
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+)
+
+func isMobile(ua string) bool {
+	isValidMobile := regexp.MustCompile(`(ios|ipod|ipad|iphone|android)`)
```

Comment:
Please move the regex definition up a level do it's not compiled on every call to these functions.

Reply:
Fixed
[commit](https://github.com/prebid/prebid-server/pull/2957/commits/cc30407f798a60b4550738f3802ffe7a19f81493)